frontend/src/components/SprintsPage/SprintsDashboard.jsx

```diff
@@ -151,24 +151,33 @@ const SprintsPage = (props) => {
               </tr>
             </thead>
             <tbody id="sprint-list-rows">
-              {allSprints.map((sprint) => (
-                <tr key={sprint.id}>
-                  <td className="sprint-list-sprint-number">{sprint.sprint}</td>
-                  <td>{sprint.start_date}</td>
-                  <td>{sprint.end_date}</td>
-                  <td>
-                    <Button
-                      id={`sprint-remove-button-${sprint.id}`}
-                      onClick={() => handleDeleteSprint(sprint.id)}
-                      className="delete-sprint-button"
-                      variant="contained"
-                      color="secondary"
-                    >
-                      Delete
-                    </Button>
-                  </td>
-                </tr>
-              ))}
+              {allSprints.sort((a, b) => b.sprint - a.sprint).map((sprint) => {
+
+                const startDateObj = new Date(sprint.start_date)
+                const endDateObj = new Date(sprint.end_date)
+
+                const formattedStartDate = `${startDateObj.getDate().toString().padStart(2, '0')}/${(startDateObj.getMonth() + 1).toString().padStart(2, '0')}/${startDateObj.getFullYear()}`
```

Comment:
You could make this a bit simpler by using:

const formattedDate = new Intl.DateTimeFormat('fi-FI').format(dateObj)

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat

Reply:
I tried first:
const formattedStartDate = new Intl.DateTimeFormat('fi-FI').format(startDateObj)

but this doesn't work directly, because standard fi-FI is DD.MM.YYYY and in form selector we have DD/MM/YYYY, and it looks silly to have two different formats in one view. 

But it is correct that my solution is not clear, I replaced with this which is much more readable: 

                const formattedStartDate = startDateObj.toLocaleDateString('fi-FI', {
                  day: '2-digit',
                  month: '2-digit',
                  year: 'numeric',
                }).replace(/\./g, '/')

Reply:
Ok, I didn't realise that!